tests/downloads/qml/CMakeLists.txt

```diff
@@ -69,9 +69,11 @@ set(QML_TESTS_LIBS
     UbuntuDownloadManager
 )
 
-find_program(XVFB xvfb)
+find_program(XVFB Xvfb)
```

Comment:
This should be `xvfb-run` instead, since it's what we're running. Checking for something else we're not using seems a bit odd.